inabox/README.md

```diff
@@ -153,6 +153,22 @@ TRACE[10-12|22:02:13.376] [batcher] AggregateSignatures took       duration=10.6
 TRACE[10-12|22:02:13.376] [batcher] Confirming batch...            caller=batcher.go:198
 ```
 
+Validate the blob was stored in a batch:
+
+```
+# Update the value of INSERT_REQUEST_ID with the result of your disperse call
+
+grpcurl -plaintext -d '{"request_id": "INSERT_REQUEST_ID"}' localhost:32003 disperser.Disperser/GetBlobStatus
```

Comment:
nit: we've been prefixing the command line code with `$` to indicate that it's meant to be executed from terminal
```
$ grpcurl -plaintext -d ...
```